docfx_project/articles/bootstrapping-trust/part-01.encryption-everywhere.md

```diff
@@ -0,0 +1,207 @@
+# Bootstrapping Trust
+
+## Part 1: Encryption Everywhere
+
+### Zero Trust
+
+Bootstrapping Trust is a series highlighting a common problem dealing
+with software systems that span multiple machines over any network. All
+networks, private (like your office/data center) or public (like the
+internet), should be considered insecure. Firewalls and segmented
+networks are not enough. Even application specific networking is not
+enough. The next step is to not trust the network.
+
+This thought process is a new way of conceptualizing network security
+and is referred to as "Zero Trust." Not that trust is not present, far
+from it. Today's networks are the home to distributed software
+solutions. Whether they are high scale mesh networks or small scale
+client servers: there are distributed software running everywhere.
+Distributed systems are rife with trust and trust issues. Zero Trust
+means: "Zero Trust Without Verification." And it turns out that putting
```

Comment:
* colons - just don't use 'em :) 
* You use "Zero Trust" to define zero trust. Instead I'd argue that "Zero Trust Without Verification" -> _something else_. no trust without verification, etc. 
* do not quote "Zero Trust Without Verification" <- this looks like a book title as it is quoted and uses capitalization as though it were a book